codecov.yml

```diff
@@ -0,0 +1,12 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: yes
+        target: auto
```

Comment:
Ah, I am reading this now: https://github.com/codecov/support/wiki/Patch-Status -- perhaps it's all good